### PR TITLE
refactor api.py

### PR DESCRIPTION
**Pull request by dianavermilya**

### CODE REVIEW

File: indicoio/utils/api.py [hunk 1]
```diff
@@ -9,36 +9,38 @@
 from indicoio import config
 
 def api_handler(arg, cloud, api, url_params=None, **kwargs):
-    if url_params is None:
-        url_params = {"api_key":None, batch:False }
-
+    """
+    Sends finalized reqeust data to ML server and receives response.
```
Comment (madisonmay) [inline indicoio/utils/api.py]: *request

File: indicoio/utils/api.py [hunk 2]
```diff
@@ -9,36 +9,38 @@
...
         raise IndicoError(error)
     return results
+
+
+def create_url(host, api, url_params):
+    api_key = url_params.get("api_key") or config.api_key
+    is_batch = url_params.get("batch")
+    apis = url_params.get("apis")
+
+    protocol_url_seg = config.url_protocol + "//%s/%s" % (host, api)
```
Comment (madisonmay) [inline indicoio/utils/api.py]: Kind of a misnomer since it's not just the , but not a big deal.

File: indicoio/utils/api.py [hunk 3]
```diff
@@ -9,36 +9,38 @@
 from indicoio import config
 
 def api_handler(arg, cloud, api, url_params=None, **kwargs):
-    if url_params is None:
-        url_params = {"api_key":None, batch:False }
-
+    """
+    Sends finalized reqeust data to ML server and receives response.
+    """
+    #setup
```
Comment (madisonmay) [inline indicoio/utils/api.py]: Comment doesn't really do anything.

**Comment by madisonmay** [comment]
Only the smallest of edits to make, as far as I'm concerned.  Looks great!

**Comment by sihrc** [comment]
lgtm :+1:

**Pull request merged**